src/ui_fsmenu/about.cc

```diff
@@ -68,7 +68,7 @@ About::About(MainMenu& fsmm, UI::UniqueWindow::Registry& r)
 			std::string label, localized_label, value, localized_value;
 		};
 		const std::vector<ContentT> content = {
-		   {"Version:", _("Version:"), format("%1$s (%2$s)", build_id(), build_type()),
+		   {"Version:", _("Version:"), format("%1$s", build_ver_details()),
 		    format(_("%1$s (%2$s)"), build_id(), build_type())},
```

Comment:
You missed the one in line 72.

Reply:
Additionally, line 71 doesn't need the `format()` call now as this statement simply copies the argument.

Reply:
So should this be

```
		const std::vector<ContentT> content = {
		   {"Version:", _("Version:"), format("%1$s", build_ver_details())},
		   {"Operating System:", _("Operating System:"),
```

Reply:
You still need 4 parameters there. I guess it should be:
```
 {"Version:", _("Version:"), build_ver_details(), format(_("%1$s"), build_ver_details())},
```

Reply:
You still need 4 parameters there. I guess it should be:
```
 {"Version:", _("Version:"), build_ver_details(), format(_("%1$s"), build_ver_details())},
```